beets/importer.py

```diff
@@ -533,6 +533,17 @@ def chosen_ident(self):
         elif self.choice_flag is action.APPLY:
             return (self.match.info.artist, self.match.info.album)
 
+    def chosen_info(self):
+        """Returns a dictionnary of metadata about the current choice.
+        May only be called when the choice flag is ASIS or RETAG
+        (in which case the data comes from the files' current metadata)
+        or APPLY (in which case the data comes from the choice).
+        """
+        if self.choice_flag in (action.ASIS, action.RETAG):
+            return self.cur_info
```

Comment:
I wonder if this might actually be easier to understand/maintain if it called `current_metadata` right here instead of using the stored field (which would avoid the need to compute `current_metadata` in the `lookup_candidates` stage).